cmd/prometheus-config-reloader/main.go

```diff
@@ -103,11 +103,18 @@ func main() {
 	reloadURL := app.Flag("reload-url", "reload URL to trigger Prometheus reload on").
 		Default("http://127.0.0.1:9090/-/reload").URL()
 
+	version.RegisterIntoKingpinFlags(app)
```

Comment:
could you also update the starting log at L146?

Reply:
fixed :)